src/containers/Layout/Sidebar.tsx

```diff
@@ -362,8 +376,82 @@ function Sidebar({ history, setSetting }: SidebarProps) {
+              fill="url(#paint0_linear_615_2035)"
+            />
+            <defs>
+              <linearGradient
+                id="paint0_linear_615_2035"
+                x1="37.2475"
+                y1="26.2516"
+                x2="-5.57766"
+                y2="-2.83053"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stopColor="#5433FF" />
+                <stop offset="0.5" stopColor="#20BDFF" />
+                <stop offset="1" stopColor="#5CFFA2" />
+              </linearGradient>
+            </defs>
+          </svg>
```

Comment:
That's Venus' logo right? We should put this in an SVG file, which can then be imported as a react component:
```
import { ReactComponent as Logo } from 'assets/img/v2/venusLogo.svg';
```

Also, I don't know if you got a chance to see the conversation we had with @coreyar yesterday, but we've settled on adding a `v2` folder inside `assets/img` to separate the assets used by v2 from the current ones. We can put the SVG file of the new logo inside there.

Reply:
sure. forgot about it while fixed in online mode